examples/migrate_server_profile_from_bay1_to_bay12.yml

```diff
@@ -0,0 +1,114 @@
+###
+# Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+- hosts: all
+  vars:
+    config: '{{ playbook_dir }}/oneview_config.json'
+    server_hardware_type_name: "SY 480 Gen9 3"
+    enclosure_group_name: "SYN107_EG"
+    server_hardware_name1: "2-USE722C99W-M, bay 1"
+    server_hardware_name2: "2-USE722C99W-M, bay 11"
+    profile_name1: "profile_san"
+    management_network: "Management"
+    SAN_A_network: "SAN-A"
+    SAN_B_network: "SAN-B"
+
+  tasks:
+
+    - name: Power Off the server hardware
+      oneview_server_hardware:
+        config: "{{ config }}"
+        state: power_state_set
+        data:
+            name : '{{ server_hardware_name1 }}'
+            powerStateData:
+                powerState: "Off"
+                powerControl: "PressAndHold"
+      delegate_to: localhost    
+
+    - name : "Create/Update Server Profile - {{ profile_name1 }}"
```

Comment:
Specify exactly what are you trying to accomplish with this task

Reply:
One caveat here though: 
Since we now do not require the server hardware to be off, the `bay1` would not even need to be declared in this file, and due to the declarative way of Ansible/Yaml this would just be listing `bayX`.

The way the example would read, would be simple and not actually be adding value. 

I'd say we should either create the original SP in this example and THEN change the blade it is using (thus illustrating the change), or drop this example.

Reply:
Ok, please review if now it makes sense (five steps in the demo folder).